v0/destinations/google_adwords_enhanced_conversions/util.js

```diff
@@ -98,24 +100,37 @@ const getConversionActionId = async (method, headers, params) => {
  * and calling the enhanced conversion.
  * data to customer list.
  * @param {*} request
+ * @param { { timeout: number } } reqOpts extra options to be set on the http client
  * @returns
  */
-const ProxyRequest = async request => {
+const ProxyRequest = async (request, reqOpts = {}) => {
   const { body, method, endpoint, params } = request;
   const { headers } = request;
+  // The reqOpts should be sent as an argument to the functions below
+  // but we might need to do a timeout/2 as there are 2 requests involved
+  const newReqOpts = {
```

Comment:
const newReqOpts = { ... reqOpts, timeout: reqOpts.timeout / 2 }
as reqOpts might have more params in the future.